### PR TITLE
add example instructions for quickstart

### PR DESCRIPTION
**Pull request by jaimergp**
just README changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,13 @@
...
 on OSX-64 right now.  To run the benchmarks, first run the download_files.py
 script, then run asv with your desired options.
+
+## Quickstart
+
+```bash
+$> conda create -n conda-benchmarks python=3.9 conda asv six pytest
+$> conda activate conda-benchmarks
+$> python download_files.py
+$> asv machine -y
```
Comment (jezdez) [inline README.md]: Can you check if `-y` really exists, that didn't work when I tried this:

```
❯ asv machine -y
usage: asv [-h] [--verbose] [--config CONFIG] [--version]
           {help,quickstart,machine,setup,run,dev,continuous,find,rm,publish,preview,profile,update,show,compare,check,gh-pages} ...
asv: error: unrecognized arguments: -y
```

File: README.md [hunk 2]
```diff
@@ -3,3 +3,13 @@
 This is what Anaconda uses to benchmark the conda project.  It is only set up
 on OSX-64 right now.  To run the benchmarks, first run the download_files.py
 script, then run asv with your desired options.
+
+## Quickstart
+
+```bash
+$> conda create -n conda-benchmarks python=3.9 conda asv six pytest
```
Comment (jezdez) [inline README.md]:
```suggestion
$> conda create -n conda-benchmarks -c conda-forge python=3.9 conda asv six pytest
```

**Review by jezdez — APPROVED** [review]
Just a few nits!

File: README.md (hunk 1 above)
Comment (jaimergp) [reply README.md]: Oops it's `--yes`.

Comment (jaimergp) [reply README.md]:
```suggestion
$> asv machine --yes
```

**Pull request merged**